### ISSUE TITLE
yolo labelling data format

### ISSUE THREAD

**Issue by Hanlin1233** [AlexeyAB/darknet#1279]
Hello, does the yolo, yolov2 and yolov3 have the same data labelling format? can I use the yolo label data to train yolov3 directly?

**Comment by zuenko** [comment]
### Yolo v2, v3
Code:
`
    size = get_img_shape(path)
    dw = 1./size[0]
    dh = 1./size[1]
    x = (x1 + x2)/2.0 # centroid x
    y = (y1 + y2)/2.0 # centroid y
    w = x2 - x1 # width
    h = y1 - y2 # height
    x = x*dw # divide by width
    w = w*dw # divide by width
    y = y*dh # divide by height
    h = h*dh # divide by height
`

Decode:
`
x1, y1 = ((x + witdth)/2)*img_width, ((y + height)/2)*img_height
x2, y2 = ((x - witdth)/2)*img_width, ((y - height)/2)*img_height
`

**Comment by zuenko** [comment]
@Hanlin1233 I'm sure about v2, v3, but you can test yolo.cfg with COCO and figure out :-)

**Comment by Hanlin1233** [comment]
@zuenko , thanks!

**Comment by TheMikeyR** [comment]
Yes the format didn't change, you can also just open the yolov1 label file and check if it have the following format per line:
classid x y w h
Where x y w h are normalized to the image size, so you are able to find the label no matter how the image have been resized.

**Comment by Hanlin1233** [comment]
@TheMikeyR , thank you very much, got it!
